src/components/RescueDetails.tsx

```diff
@@ -275,6 +307,14 @@ export default function RescueDetails({id}: { id: string }) {
         }
     }
 
+    function renderSecondVolunteerElements(rescue : BirdAlert) {
+        if (rescue.secondVolunteer && rescue.twoPersonRescue) {
```

Comment:
you could just return early if `!rescue.twoPersonRescue`

Reply:
Good catch, will change that.